src/CMakeLists.txt

```diff
@@ -103,3 +108,26 @@ install(TARGETS umf
 )
 
 add_subdirectory(pool)
+
+# TODO: enable proxy_lib on Windows
+if(LINUX)
+    if(UMF_PROXY_LIB_BASED_ON_POOL STREQUAL SCALABLE)
+        set(PROXY_LIB_USES_SCALABLE_POOL ON)
+        set(PROXY_LIBS umf scalable_pool)
+        if(UMF_BUILD_LIBUMF_POOL_SCALABLE)
+            add_subdirectory(proxy_lib)
+        else()
+            message(WARNING "Disabling the proxy library, because UMF_PROXY_LIB_BASED_ON_POOL==SCALABLE but UMF_BUILD_LIBUMF_POOL_SCALABLE is OFF")
+        endif()
+    elseif(UMF_PROXY_LIB_BASED_ON_POOL STREQUAL JEMALLOC)
+        set(PROXY_LIB_USES_JEMALLOC_POOL ON)
+        set(PROXY_LIBS umf jemalloc_pool)
+        if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
+            add_subdirectory(proxy_lib)
+        else()
+            message(WARNING "Disabling the proxy library, because UMF_PROXY_LIB_BASED_ON_POOL==JEMALLOC but UMF_BUILD_LIBUMF_POOL_JEMALLOC is OFF")
+        endif()
+    else()
+        message(FATAL_ERROR "Proxy library: pool manager not chosen")
```

Comment:
pls add info that it should be set via `UMF_PROXY_LIB_BASED_ON_POOL`.

Reply:
Done